pkg/ingress/tls.go

```diff
@@ -128,5 +129,11 @@ func (i *Tls) RequestCert() error {
 
 	s.Object().Data = certData
 
-	return s.Save()
+	fmt.Printf("secret object data: %v\n", s)
+
+	returnVal := s.Save()
+
+	fmt.Printf("secret save error: %v\n", returnVal)
```

Comment:
@gianrubio How should I test it? The `..Debugf()` functions `%v` doesn't have a problem with nil value and others.